src/ProjectNeon/Assets/Features/Battle/Cards/EnemyCardSelection.cs

```diff
@@ -8,17 +8,10 @@ public class EnemyCardSelection : MonoBehaviour
     [SerializeField] private GameEvent onEnemyTurnsConfirmed;
     [SerializeField] private BattleState battle;
```

Comment:
This might be better solved if we give this Script access to the `EnemyArea` for the purposes of letting the Enemies take their actions.

Reply:
@SilasReinagel I agree, this is just an initial implementation. Probably we will have a similar construct to player moves, so maybe we will be able to reuse them